tools/clang/lib/SPIRV/SPIRVEmitter.cpp

```diff
@@ -4873,7 +4904,8 @@ uint32_t SPIRVEmitter::processIntrinsicAllOrAny(const CallExpr *callExpr,
                                     &matColCount)) {
       if (!elemType->isFloatingType()) {
         emitError("'all' and 'any' currently do not take non-floating point "
-                  "matrices as argument.");
+                  "matrices as argument",
```

Comment:
non-floating-point arguments in all/any intrinsic function unimplemented

Reply:
Done